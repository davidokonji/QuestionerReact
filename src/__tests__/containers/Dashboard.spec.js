import { shallow } from 'enzyme';
import React from 'react';
import Dashboard from '../../containers/Dashboard.container';

describe('Dashboard container', () => {
  it('should  render dashboard container', () => {
    const wrapper = shallow(<Dashboard />);

    expect(wrapper).toMatchSnapshot();
  });
});
