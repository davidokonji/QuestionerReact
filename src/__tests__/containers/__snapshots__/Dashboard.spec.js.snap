// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard container should  render dashboard container 1`] = `
<Fragment>
  <ConnectFunction />
  <div
    className=""
  >
    <ConnectFunction />
  </div>
  <Footer />
</Fragment>
`;
